components/AboutUs.js

```diff
@@ -56,7 +92,9 @@ const About = () => {
           today and start your journey to finding love. We promise it&apos;ll be
```

Comment:
this is just a note for the future because I completely forgot about this until my last PR, but in vercel you have to use html entities for apostrophe's/quotes or it will fail to deploy - not 100% sure why but just something to keep in mind when adding new text!